src/contracts/AccountingEngine.sol

```diff
@@ -213,12 +214,12 @@ contract AccountingEngine is Authorizable, Modifiable, Disableable, IAccountingE
     // auction surplus percentage
     if (_params.surplusTransferPercentage < ONE_HUNDRED_WAD) {
       _id = surplusAuctionHouse.startAuction({
-        _amountToSell: _params.surplusAmount.wmul(ONE_HUNDRED_WAD - _params.surplusTransferPercentage),
+        _amountToSell: wadToRad(_params.surplusAmount.wmul(ONE_HUNDRED_WAD - _params.surplusTransferPercentage)),
```

Comment:
Basically we were getting overflows during calculations because of the size of RADs.  I think I took this in an unnecessary direction.

Weis3r has actually made some other suggestions that I think equate to a better solution for this ... which is that we leave everything in RAD.  wmul will work for all values where `surplusAmount.wmul(WAD - surplusTransferPercentage) < max(uint256)`

It's not you that is confused; I confused myself a little bit on this one 🤣